src/Server.h

```diff
@@ -71,6 +72,20 @@ class cServer
 	int  GetNumPlayers(void) const;
 	void SetMaxPlayers(int a_MaxPlayers) { m_MaxPlayers = a_MaxPlayers; }
 
+	// tolua_end
+	/** Add a Forge mod to the server ping list. */
+	bool RegisterForgeMod(AString & a_ModName, AString & a_ModVersion);
+
+	/** Add a Forge mod to the server ping list for one protocol version. */
+	bool RegisterForgeModForProtocol(AString & a_ModName, AString & a_ModVersion, UInt32 a_ProtocolVersionNumber);
+	// tolua_begin
+
+	/** Remove a Forge mod from the server ping list. */
+	void UnregisterForgeMod(AString & a_ModName);
+
+	/** Remove a Forge mod to the server ping list for one protocol version. */
+	void UnregisterForgeModForProtocol(AString & a_ModName, UInt32 a_ProtocolVersionNumber);
+
```

Comment:
Now you removed the following functions from the API. `// tolua_begin` missing here.